Meta/serenity.sh

```diff
@@ -156,9 +156,11 @@ is_supported_compiler() {
     if $COMPILER --version 2>&1 | grep "Apple clang" >/dev/null; then
         return 1
     elif $COMPILER --version 2>&1 | grep "clang" >/dev/null; then
-        [ "$MAJOR_VERSION" -gt 12 ] && return 0
+        # Clang version check
+        [ "$MAJOR_VERSION" -gt 13 ] && return 0
     else
-        [ "$MAJOR_VERSION" -gt 10 ] && return 0
+        # GCC version check
+        [ "$MAJOR_VERSION" -gt 12 ] && return 0
     fi
```

Comment:
GCC 13 isn't even out though... this should be `-gt 11` (or `-ge 12`).

Reply:
oops. good point. i'll make them -ge 13 and -ge 12 then.